c/tskit/trees.h

```diff
@@ -54,6 +54,9 @@ extern "C" {
 #define TSK_STAT_ALLOW_TIME_UNCALIBRATED (1 << 12)
 #define TSK_STAT_PAIR_NORMALISE          (1 << 13)
 
+/* Stat-specific options */
+#define TSK_STAT_NONCENTRED         (1 << 12)
```

Comment:
Why not 1 << 14? We could pass both TSK_STAT_NONCENTRED  and TSK_STAT_ALLOW_TIME_UNCALIBRATED  to the stats API, couldn't we?

Reply:
yes, not sure what I was thinking here